compiler/util/misc.cpp

```diff
@@ -368,6 +372,7 @@ static bool printErrorHeader(BaseAST* ast) {
 
           err_fn = fn;
         }
+        err_fn_id = err_fn->id;
```

Comment:
Would be nice to add a brief explanation of your changes to misc.cpp to the RP message.